backend/src/main/kotlin/be/osoc/team1/backend/services/ProjectService.kt

```diff
@@ -98,20 +99,21 @@ class ProjectService(
      * Gets conflicts (a conflict involves a student being assigned to 2 projects at the same time)
      */
     fun getConflicts(edition: String): MutableList<Conflict> {
-        val baseUrl = ServletUriComponentsBuilder.fromCurrentContextPath().build().toUriString()
-        val studentsMap = mutableMapOf<UUID, MutableList<String>>()
+        val studentsMap = mutableMapOf<Student, MutableList<String>>()
+        val projectSerializer = ProjectSerializer()
         for (project in getAllProjects(edition)) {
             for (student in getStudents(project)) {
                 // add project id to map with student as key
-                studentsMap.putIfAbsent(student.id, mutableListOf())
-                studentsMap[student.id]!!.add("$baseUrl/projects/" + project.id)
+                studentsMap.putIfAbsent(student, mutableListOf())
+                studentsMap[student]!!.add(projectSerializer.toUrl(project))
             }
         }
         val conflicts = mutableListOf<Conflict>()
-        for ((studentId, projectIds) in studentsMap.entries) {
+        val studentSerializer = StudentSerializer()
+        for ((student, projectIds) in studentsMap.entries) {
```

Comment:
I know this isn't your fault, but could you change projectIds to ProjectURLs? That way the parameter is more descriptive.